File: frameworks/python/fastapi/Implementation/routes/baseline.py
from fastapi import APIRouter
from fastapi.responses import PlainTextResponse

from payloads import Payloads


def router(p: Payloads) -> APIRouter:
    """baseline: the dispatch floor, with nothing serialised."""
    routes = APIRouter()

    @routes.get("/plaintext", response_class=PlainTextResponse)
    async def plaintext() -> str:
        return "Hello, World!"

    return routes

File: frameworks/node/koa/Implementation/routes/baseline.ts
import type { Routes } from "../app.ts";

/** baseline: the dispatch floor, with nothing serialised. Koa sends a string that does not start with `<` as text/plain. */
const baseline: Routes = (router) => {
  router.get("/plaintext", (ctx) => {
    ctx.body = "Hello, World!";
  });
};

export default baseline;
